features/support/env.rb

```diff
@@ -74,4 +86,6 @@
 Before do
   Devise.mailer.deliveries = []
   page.driver.browser.manage.window.resize_to(1024, 500)
+  # Delete all files in "~/diaspora/tmp/downloads"
```

Comment:
I find the example path in the comments a bit confusing, IMO just strip the `~/diaspora/` from them :)